app/routes/events/view/export.js

```diff
@@ -5,5 +5,8 @@ const { Route } = Ember;
 export default Route.extend({
   titleToken() {
     return this.l10n.t('Export');
+  },
+  model() {
+    return this.modelFor('events.view');
```

Comment:
Not needed I guess. The parent model should be accessible if there is no model method defined here